package.json

```diff
@@ -24,7 +24,7 @@
     "prepare": "is-ci || husky install",
     "pnpm:devPreinstall": "is-ci || node ./scripts/doctor.js",
     "test:example": "pnpm --filter \"example-*\" build",
-    "test:unit": "NO_COLOR=1 pnpm --filter \"@rspack/*\" test",
+    "test:unit": "cross-env NO_COLOR=1 pnpm --filter \"@rspack/*\" test",
```

Comment:
Could you move `NO_COLOR=1` to `packages/rspack/package.json`, some people (me) will use that command to run test